api/lib/validator-schemas.js

```diff
@@ -0,0 +1,35 @@
+const { gql } = require('apollo-server')
+const validatorTypeDefs = gql`
+  enum ContributionType {
+    ARTICLE
+    WEBSITE
+    PODCAST
+    GITHUB
+  }
+
+  type ProfileLink {
```

Comment:
```suggestion
  type ProfileLinks {
```

Reply:
type are always singular, if you look around in schema. GQL populates them for us

Reply:
yeah but this is not really a `type` it's a group of links

Reply:
ah true xD you are right